demo-ui/app/uk/gov/ons/addressIndex/demoui/views/singleMatch.scala.html

```diff
@@ -27,6 +29,44 @@ <h3 class="panel-title">@messages("single.sfatext")</h3>
                 </div>
                 }
             </div>
+            @addressBySearchResponse.map { aresponse =>
+            <div class="panel panel-single address-panel">
+                <h3 class="green">@aresponse.total addresses found</h3>
+            </div>
+            @aresponse.addresses.map{ address =>
+            <div class="panel panel-match address-panel">
+                <div class="result-content">
+                    <div class="alert alert-success" role="alert">
+                        <table class="matchResults">
+                            <tr>
+                                <td class="match-found-txt"><strong>Residential</strong></td>
+                                <td style="text-align:right" class="match-score">Matching Score: @address.underlyingScore</td>
```

Comment:
Optional: You can visually improve this by dividing this number by the maxScore, multiplying it by 100 and naming it "confidence" (also colors (red, yellow, green) could be nice)